src/ScriptBuilder/CSharpCounterWriter.cs

```diff
@@ -13,23 +13,28 @@ public static void WriteCode(string scriptPath, IEnumerable<DurationDefinition>
             {
                 var stringBuilder = new StringBuilder();
 
-                var slaCounterDefinition = @"new CounterCreationData(""SLA violation countdown"", ""Seconds until the SLA for this endpoint is breached."", PerformanceCounterType.NumberOfItems32),";
+                var slaCounterDefinition = $@"new CounterCreationData(""SLA violation countdown"", ""{CounterNameAndDescriptionConventions.LegacySlaViolationCounterDescription}"", PerformanceCounterType.NumberOfItems32),";
                 stringBuilder.AppendLine(slaCounterDefinition.PadLeft(slaCounterDefinition.Length + 8));
 
                 foreach (var duration in durations)
                 {
                     var averageTimerName = duration.Name.GetAverageTimerCounterName();
                     var averageTimerBase = duration.Name.GetAverageTimerBaseCounterName();
+                    var averateTimerDescription = CounterNameAndDescriptionConventions.GetAverageTimerDescription(duration.Name, duration.Description);
 
-                    var durationAverageDefinition = $@"new CounterCreationData(""{averageTimerName}"", ""{duration.Description}"", PerformanceCounterType.AverageTimer32),";
+                    var durationAverageDefinition = $@"new CounterCreationData(""{averageTimerName}"", ""{averateTimerDescription}"", PerformanceCounterType.AverageTimer32),";
```

Comment:
I think this is not correct @ramonsmits . There's no `AverageTimer64` (there's a counter 64, but not the timer). I also tried to break the timer reporting 1h periods using [the following gist](https://gist.github.com/Scooletz/5c53ec8fce7854023097ce4c5e0ec000). It works perfectly showing 1h in PerfMon (scaled ofc).

<img width="590" alt="timer_for_1h" src="https://user-images.githubusercontent.com/519707/29405269-b945b93e-833d-11e7-89be-350a65d2193b.png">